edb/server/defines.py

```diff
@@ -34,7 +34,7 @@
 EDGEDB_SPECIAL_DBS = {EDGEDB_TEMPLATE_DB, EDGEDB_SYSTEM_DB}
 
 # Increment this whenever the database layout or stdlib changes.
-EDGEDB_CATALOG_VERSION = 2021_03_05_00_00
+EDGEDB_CATALOG_VERSION = 2021_03_15_00_00
```

Comment:
This probably means change can't be backported, right? :(

Reply:
Yes, there's no way around this I'm afraid.